ui/helpers/constants/settings.js

```diff
@@ -505,16 +505,13 @@ const SETTINGS_CONSTANTS = [
     route: `${DEVELOPER_OPTIONS_ROUTE}#service-worker-keep-alive`,
     icon: IconName.CodeCircle,
   },
-];
-
-if (process.env.ENABLE_CONFIRMATION_REDESIGN) {
-  SETTINGS_CONSTANTS.push({
+  {
     tabMessage: (t) => t('experimental'),
     sectionMessage: (t) => t('redesignedConfirmationsEnabledToggle'),
     descriptionMessage: (t) => t('redesignedConfirmationsToggleDescription'),
     route: `${EXPERIMENTAL_ROUTE}#redesigned-confirmations`,
     icon: 'fas fa-flask',
-  });
-}
+  },
+];
```

Comment:
[in and partially out of scope]

would be nice to move these above the snaps experimental settings. We should update the references as well. Looks like the snaps one has gotten out of order. With this change, we should have 

```
  // experimental settingsRefs[0]
  {
    tabMessage: (t) => t('experimental'),
    sectionMessage: (t) => t('petnamesEnabledToggle'),
    descriptionMessage: (t) => t('petnamesEnabledToggleDescription'),
    route: `${EXPERIMENTAL_ROUTE}#nicknames`,
    icon: 'fas fa-flask',
  },
  // experimental settingsRefs[1]
  {
    tabMessage: (t) => t('experimental'),
    sectionMessage: (t) => t('notificationsFeatureToggle'),
    descriptionMessage: (t) => t('notificationsFeatureToggleDescription'),
    route: `${EXPERIMENTAL_ROUTE}#notifications`,
    icon: 'fas fa-flask',
  },
  // experimental settingsRefs[2]
  {
    tabMessage: (t) => t('experimental'),
    sectionMessage: (t) => t('redesignedConfirmationsEnabledToggle'),
    descriptionMessage: (t) => t('redesignedConfirmationsToggleDescription'),
    route: `${EXPERIMENTAL_ROUTE}#redesigned-confirmations`,
    icon: 'fas fa-flask',
  },
  ///: BEGIN:ONLY_INCLUDE_IF(keyring-snaps)
  // since this route is only included with keyring-snaps feature flag, this needs to be the last settingsRef for the experimental tab
  // experimental settingsRefs[3]
  {
    tabMessage: (t) => t('experimental'),
    sectionMessage: (t) => t('snaps'),
    descriptionMessage: (t) => t('addSnapAccountToggle'),
    route: `${EXPERIMENTAL_ROUTE}#snaps`,
    icon: 'fas fa-flask',
  },
```

then the refs in `ui/pages/settings/experimental-tab/experimental-tab.component.js` should be updated appropriately